MAUI/TabView/Tab-Item-Customization.md

```diff
@@ -478,4 +478,24 @@ tabView.IsScrollButtonEnabled = true;
 
 {% endtabs %}
 
-![TabView Scroll Mode](images/TabViewScroll.gif) 
\ No newline at end of file
+![TabView Scroll Mode](images/TabViewScroll.gif) 
+
+## Content transition duration
+
+We can customize the duration of the animation when changing the selected index of the SelectionIndicator in the Tab View by setting the `ContentTransitionDuration` property.
```

Comment:
You can customize the animation duration when changing the selected index of the Tab View by setting the `ContentTransitionDuration` property.

Reply:
Added Changes